tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -1619,6 +1619,19 @@ def map(self, map_func, num_parallel_calls=None, deterministic=None):
     >>> list(d.as_numpy_iterator())
     [b'HELLO', b'WORLD']
 
+    3) Use `tf.numpy_function`, which also allows you to write arbitrary
```

Comment:
Please add information about that fact the the use of `tf.numpy_function` and `tf.py_function` in general precludes the possibility executing user-defined transformations in parallel (because of Python GIL).

Reply:
Done